beacon-chain/rpc/beaconv1/state.go

```diff
@@ -246,6 +255,9 @@ func (bs *Server) justifiedStateRoot(ctx context.Context) ([]byte, error) {
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not get justified checkpoint: %v", err)
 	}
+	if cp == nil {
+		return nil, status.Errorf(codes.NotFound, "No justified checkpoint exists")
+	}
```

Comment:
No test for this code as it's impossible to save a state with empty checkpoint in the DB.